test/sqllogictest/cluster.slt

```diff
@@ -122,7 +122,7 @@ statement ok
 CREATE DEFAULT INDEX foo_v_idx IN CLUSTER foo ON v
 
 query TTTT
-SHOW INDEXES IN CLUSTER bar;
+SHOW INDEXES IN CLUSTER bar WHERE name NOT LIKE 'mz_%';
```

Comment:
Can you also add a test with `set cluster = bar; show indexes` that would test that we omit the system indexes if no clauses are specified? Or is that already tested somewhere?

Reply:
That's tested here: https://github.com/MaterializeInc/materialize/blob/06b726fb743af87811d58321e657e4438da0f963/test/testdrive/indexes.td#L245-L246